hdl/Encoding8b10b.bsv

```diff
@@ -47,22 +68,26 @@ function Bool is_d(Value v) =
 function Bool is_k(Value v) = !is_d(v);
 
 //
-// Character
+// `Character`
 //
-// A type alias representing a 10 bit encoded value, often referred to as a character. Note: while
-// not enforced in any way, these characters are often MSB first.
+// A type representing a 10 bit encoded value, sometimes referred to as a
+// character. Note: while not enforced in any way, these characters are
+// transmitted MSB first.
```

Comment:
I think this is that they are *represented* as MSB first, not *transmitted* MSB first right? All 8b10b transmission should be done LSB first, otherwise you can violate comma-character uniqueness (a fake comma can be seen across two symbol/character boundaries if they are transmitted MSB first)

Reply:
Yes, my comment here was incorrect. Both the `encode` and `decode` functions expect these to be in `jhgf_iedcba` bit order and this is reflected in the description of the `Character` type and `mk_c` function.